resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -378,13 +378,16 @@ private[spark] object Config extends Logging {
   val KUBERNETES_VOLUMES_HOSTPATH_TYPE = "hostPath"
   val KUBERNETES_VOLUMES_PVC_TYPE = "persistentVolumeClaim"
   val KUBERNETES_VOLUMES_EMPTYDIR_TYPE = "emptyDir"
+  val KUBERNETES_VOLUMES_NFS_TYPE = "nfs"
```

Comment:
seems like  full name is prefeered: NetworkFileSystem

Reply:
I think they are supposed to be consistent with the volume type names used by Kubernetes?
https://kubernetes.io/docs/concepts/storage/volumes/#types-of-volumes